cmd/limactl/main.go

```diff
@@ -23,10 +25,30 @@ func main() {
 }
 
 func newApp() *cobra.Command {
+	examplesDir := "$PREFIX/share/doc/lima/examples"
+	if exe, err := os.Executable(); err == nil {
+		binDir := filepath.Dir(exe)
+		prefixDir := filepath.Dir(binDir)
+		examplesDir = filepath.Join(prefixDir, "share/doc/lima/examples")
+	}
+
 	var rootCmd = &cobra.Command{
-		Use:           "limactl",
-		Short:         "Lima: Linux virtual machines",
-		Version:       strings.TrimPrefix(version.Version, "v"),
+		Use:     "limactl",
+		Short:   "Lima: Linux virtual machines",
+		Version: strings.TrimPrefix(version.Version, "v"),
+		Example: fmt.Sprintf(`  Start the default instance:
+  $ limactl start
+
+  Open a shell:
+  $ lima
+
+  Run a container:
+  $ lima nerdctl run -d --name nginx -p 8080:80 nginx:alpine
+
+  Stop the default instance:
+  $ limactl stop
```

Comment:
This does not work right now, as the stop command has `cobra.ExactArgs(1)`. So you have to write `limactl stop ""`. This should be fixed by using `cobra.MaximumNArgs(1)` instead.

Reply:
fixed